### PR TITLE
enh(ui): update style of events view page

### PR DESCRIPTION
**Pull request by kduret**
## Description

Listing is more dense : 
image

**Fixes** MON-5016

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

check new style

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@centreon/ui": "github:centreon/centreon-ui#adbb81c3b4dc6bf67842b890c2ac4e175562decd",
+    "@centreon/ui": "github:centreon/centreon-ui#MON-5016-style",
```
Comment (bdauria) [inline package.json]: We need to be careful to merge the corresponding centreon-ui branch before merging this one to avoid breaking the build afterwards

**Comment by sc979** [comment]
The screenshot seems much more user friendly

File: www/front_src/src/Resources/columns/Graph/index.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ import { ColumnProps } from '..';
 const JSXXAxis = (XAxis as unknown) as (props) => JSX.Element;
 const JSXYAxis = (YAxis as unknown) as (props) => JSX.Element;
 
-const graphHeight = 350;
+const graphHeight = 200;
```
Comment (bdauria) [inline www/front_src/src/Resources/columns/Graph/index.tsx]: 200 is too small as a height for the graph, the legend can overlap if there are too many elements:

Screenshot from 2020-03-29 11-27-56

Comment (kduret) [reply www/front_src/src/Resources/columns/Graph/index.tsx]: I think the legend can overlap also with 350 height (eg: monitoring of network interfaces of a switch)
It should be better to make a custom legend to manage lot of metrics

Comment (kduret) [reply www/front_src/src/Resources/columns/Graph/index.tsx]: Anyway, I've increased the height to 250

**Comment by kduret** [comment]
> @kduret the padding is too small in two areas: between the action icons and within the table rows. I think it's better to add some more padding (at least one unit), and reduce the size of the icons instead (Status & State chips)

I move cell padding to 0.5 (instead of 0.25) cause 1 was too big, even if I reduce the size of the icons.
I've also added some padding between actions icons

**Pull request merged**